.github/workflows/release.yml

```diff
@@ -3,17 +3,36 @@ name: 🎉 Release
 on:
   push:
     tags:
-      - "*" # Trigger on all tags
+      - "*"
 
 jobs:
   release:
     name: Release
     uses: ./.github/workflows/reusable-build.yml
     with:
       goreleaser_config: goreleaser.yml
-      goreleaser_options: "--clean"
+      goreleaser_options: ""
```

Comment:
Why remove this?

Reply:
I copy and paste error from the webscan repo where I got the file. I will add it back